.drone.star

```diff
@@ -1693,10 +1710,9 @@ def ocisServer(storage, accounts_hash_difficulty = 4, volumes = [], depends_on =
             "PROXY_ENABLE_BASIC_AUTH": True,
             "WEB_UI_CONFIG": "/drone/src/tests/config/drone/ocis-config.json",
             "IDP_IDENTIFIER_REGISTRATION_CONF": "/drone/src/tests/config/drone/identifier-registration.yml",
-            "OCIS_LOG_LEVEL": "error",
+            "OCIS_LOG_LEVEL": "debug",
```

Comment:
Is that really necessary? In the past we had real issues with so many log lines hammering our storage system.

Reply:
the problem is also that drone truncated the log lines, so you might have trouble to see the first ones

Reply:
yes, I think that drone truncates pipeline step output at around 64k lines! I don't remember if you see the first or last 64k.

Reply:
Yeah that line is a left over from debugging a test failure that I couldn't reproduce locally. Removed it now.